etc/config-template.toml

```diff
@@ -142,6 +142,9 @@
 # Use delete range to drop a large number of continuous keys.
 # use-delete-range = false
 
+# delay time before deleting a region range
```

Comment:
Please add config for coprocessor request timeout, range-deletion-delay relay on it.